parquet/src/main/scala/magnolify/parquet/ParquetType.scala

```diff
@@ -54,10 +54,16 @@ object ParquetArray {
   }
 }
 
+class doc(doc: String) extends StaticAnnotation with Serializable {
+  override def toString: String = doc
+}
```

Comment:
I start thinking we should define this in the `shared` module and only create a type alias in the other packages.

Reply:
Done